builtins/web/timers.cpp

```diff
@@ -100,22 +121,20 @@ class TimerTask final : public api::AsyncTask {
     for (auto &arg : arguments_) {
       TraceEdge(trc, &arg, "Timer callback arguments");
     }
+    TIMERS_MAP->trace(trc);
```

Comment:
While this works, it means that we call `TIMERS_MAP->trace` from every timer, while that method itself traces through the timers. Also, with the `PersistentRooted` thing above, it's not needed anymore.
```suggestion
```